tests/test_user.py

```diff
@@ -56,29 +56,36 @@ def login(self):
       self.session_token = response['session']['session_token']
 
   def post(self, url, data=None):
-    header = {"Authorization": "Bearer {}".format(self.session_token)}
+    header = self.init_header()
     if data is None:
       response = self.test_client.post(url, headers=header)
     else:
       response = self.test_client.post(url, headers=header, data=data)
     return response
 
   def get(self, url, data=None):
-    header = {"Authorization": "Bearer {}".format(self.session_token)}
+    header = self.init_header()
     if data is None:
       response = self.test_client.get(url, headers=header)
     else:
       response = self.test_client.get(url, headers=header, data=data)
     return response
 
   def delete(self, url, data=None):
-    header = {"Authorization": "Bearer {}".format(self.session_token)}
+    header = self.init_header()
     if data is None:
       response = self.test_client.delete(url, headers=header)
     else:
       response = self.test_client.delete(url, headers=header, data=data)
     return response
 
+  def init_header(self):
```

Comment:
Not too critical but i should probably not pass these in by default and instead take in some opt_header arg where the test user can specify if he needs additional headers other than the session info

Reply:
yeah that would work, off the top of my head I can't think of a reason why someone using a facebook user wouldn't want to supply the access token, besides I guess testing that the token is needed. Either way I think what you have is good for now.